kotest-framework/kotest-framework-plugin-gradle/src/main/kotlin/io/kotest/framework/gradle/KotestPlugin.kt

```diff
@@ -78,24 +98,21 @@ open class KotestPlugin : Plugin<Project> {
                val capitalTarget = name.replaceFirstChar { it.uppercase() }
                // gradle best practice is to only apply to this project, and users add the plugin to each subproject
                // see https://docs.gradle.org/current/userguide/isolated_projects.html
-               project.tasks.register("kotest$capitalTarget", KotestAndroidTask::class.java) {
+               project.tasks.register("kotest$capitalTarget", KotestAndroidTask::class) {
                   description = DESCRIPTION
-                  group = JavaBasePlugin.VERIFICATION_GROUP
```

Comment:
how come the group was removed ?

Reply:
Because it was applicable to all tasks I moved it to a `configureEach {}` block above.

Reply:
gotcha